frontend/fingertips-frontend/app/search/results/search-result-data.ts

```diff
@@ -0,0 +1,36 @@
+export interface SearchResultInterface {
```

Comment:
renaming the interface causes a naming conflict within `SearchResult` component. This could be solved using an import alias, so I'm fine with whichever approach is chosen.

Reply:
Ah. I think if there's already a component called `SearchResult`, then having this type named `SearchResultInterface` might be even more confusing!

How about, since it's the result of searching for an indicator `IndicatorSearchResult`? It's looking likely that we will be searching for things other than indicators in future, so it's probably worth being clear what kind of result it is.